src/main/java/org/zalando/nakadi/controller/CursorsController.java

```diff
@@ -83,7 +82,7 @@ public CursorsController(final CursorsService cursorsService,
         try {
             final List<SubscriptionCursor> cursors = cursorsService.getSubscriptionCursors(subscriptionId, client)
                     .stream()
-                    .map(cursor -> cursorConverter.convert(cursor, cursorTokenService.generateToken()))
+                    .map(cursor -> cursor.withToken(cursorTokenService.generateToken()))
```

Comment:
@antban @adyach maybe something like `addToken(final String token)`, or enhanceWithToken(final String token)`, or perhaps getSubscriptionCursorWithToken(final String token)`, or even shortened to `getWithToken(final String token)`. There's also `setToken(final String token)`, but that looks strange in an Immutable class